api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -259,10 +267,11 @@ private RdrWorkspace toRdrWorkspace(DbWorkspace dbWorkspace) {
     Map<String, FirecloudWorkspaceAccessEntry> aclMap = firecloudResponse.getAcl();
     aclMap.forEach(
         (email, access) -> {
-          RdrWorkspaceUser workspaceUderMap = new RdrWorkspaceUser();
-          workspaceUderMap.setUserId((int) userDao.findUserByUsername(email).getUserId());
-          workspaceUderMap.setRole(RdrWorkspaceUser.RoleEnum.fromValue(access.getAccessLevel()));
-          rdrWorkspace.addWorkspaceUsersItem(workspaceUderMap);
+          RdrWorkspaceUser workspaceUserMap = new RdrWorkspaceUser();
+          workspaceUserMap.setUserId((int) userDao.findUserByUsername(email).getUserId());
+          workspaceUserMap.setRole(RdrWorkspaceUser.RoleEnum.fromValue(access.getAccessLevel()));
+          workspaceUserMap.setStatus(RdrWorkspaceUser.StatusEnum.fromValue("ACTIVE"));
```

Comment:
why is this being set now?

Reply:
This has been made required for user by RDR. this is NOT part of this PR it just got sneaked in. I can revert this and make a new PR for this

Reply:
I think that may be better even if its a small change since I have a lingering comment about its implementation.

That comment being, how do we know this value can be set to "ACTIVE"? IIRC, firecloud can return a value for the NO_ACCESS role.

Reply:
Thats because as of now for RDR we are going with assumption that all the USERS are active until we introduce an **official** way of removing users from workbench

Reply:
I see. It does seem a bit odd that we would send over "ACTIVE" for disabled users but if that's what they want ¯\_(ツ)_/¯.

Maybe add a comment saying we should set this dynamically when we do start removing users from workbench?

Reply:
Is there a separate notion of `RDR_ACTIVE` from `WORKBENCH_ACTIVE`?